frontend/src/components/parties/RelationshipProfile.js

```diff
@@ -47,6 +48,17 @@ const mapPermitGuidToNumber = (permits) =>
     return acc;
   }, {});
 
+const mapTSFGuidToName = (tailings) => {
+  const tail = tailings.reduce(
+    (acc, { mine_tailings_storage_facility_guid, mine_tailings_storage_facility_name }) => {
+      acc[mine_tailings_storage_facility_guid] = mine_tailings_storage_facility_name;
+      return acc;
+    },
+    {}
+  );
+  return tail;
```

Comment:
trivial but you can remove the const tail, return tail and have the arrow function point to tailings.reduce directly.

const mapTSFToName = (tailings) => tailings.reduce();

Reply:
I removed this it wasn't working for me yesterday for some reason and this was the only way I was able to get it to work. Seems to be good now.

Reply:
Nice!